src/webui/src/components/overview/command/Command2.tsx

```diff
@@ -10,7 +10,7 @@ export const Command2 = (): any => {
 
     if (clusterMetaData !== undefined) {
         for (const item of clusterMetaData) {
-            if (item.key === 'command') {
+            if (item.key === 'command' || item.key === 'trial_config') {
                 trialCommand = item.value;
```

Comment:
the data format in trial_config is {"command":"python3..", "gpuNum":"",...}, should use item.value.command for trial_config.

Reply:
will fix